### PR TITLE
Use AVA instead of tape and return a PassThrough stream

### PR DESCRIPTION
**Pull request by kevva**
Seems like https://github.com/sindresorhus/unzip-response/commit/c434b793974e42e8011ddcabdb408c33872bf5f8 was a false positive. When implementing `ava` I noticed `get-stream` still threw the unexpected EOF error.

Tested this with `got` too and all tests pass.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,8 +37,10 @@
     "stream"
   ],
   "devDependencies": {
+    "ava": "*",
     "get-stream": "^2.3.0",
-    "tape": "^4.0.0",
+    "pify": "^2.3.0",
+    "rfpify": "^1.0.0",
     "xo": "*"
```
Comment (SamVerschueren) [inline package.json]: Add esnext flag

**Comment by kevva** [comment]
Btw, I could split up this PR into two since they are somewhat unrelated if you want me to.

File: package.json (hunk 1 above)
Comment (kevva) [inline package.json]: Forgot about `clinton`, thanks!

**Comment by sindresorhus** [comment]
@kevva Can you add a test to `got` too? Just to ensure everything works.

File: index.js [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
+			stream.emit('error', err);
 		});
 
-		res.on('close', () => {
-			unzip.emit('close');
-		});
+		res.pipe(unzip).pipe(stream);
 
-		res = res.pipe(unzip);
+		res = stream;
```
Comment (sindresorhus) [inline index.js]: Would be better if we just returned `stream` below instead of overwriting `res`.

**Comment by kevva** [comment]
> @kevva Can you add a test to got too? Just to ensure everything works.

https://github.com/sindresorhus/got/commit/e25457da16a6dc74656945bd54cf39d4b3be9a5c

**Comment by sindresorhus** [comment]
Great work @kevva. Node.js streams continue to be a complete mystery to me.

**Comment by SamVerschueren** [comment]
> Node.js streams continue to be a complete mystery to me.

I thought I was the only one...

**Pull request merged**